src/content/docs/integrations/opensearch.mdx

```diff
@@ -39,3 +41,40 @@ to "opensearch://localhost:9200", id=doc_id, doc=event, action="update", index=c
 
 The above example updates the document with ID `XXX` with the contents from the
 nested field `event`.
+
+### Accept data by emulating OpenSearch
+
+Tenzir can act as drop-in replacement for OpenSearch by accepting data via a
+Bulk API endpoint. This allows you to point your
+[Logstash](https://opensearch.org/docs/latest/tools/logstash/index/) or Beats
+instances to Tenzir instead.
+
+```tql
+from "opensearch://localhost:9200"
+publish "opensearch"
+```
+
+This pipeline accepts data on port 9200 and publishes all received events on the
+`opensearch` topic for further processing by other pipelines.
+
+#### Ship data via Logstash
+
+To send data to the OpenSearch endpoint, use the standard Logstash configuration
+with the OpenSearch output plugin. Here's an example of your
+`pipeline.conf`:
+
+```javascript
+output {
+  opensearch {
+    hosts       => "https://localhost:9200"
+    user        => "admin"
+    password    => "admin"
+    index       => "logstash-logs-%{+YYYY.MM.dd}"
+    ssl_certificate_verification => false
+  }
+}
```

Comment:
```suggestion
output {
  opensearch {
    hosts       => "http://localhost:9200"
    index       => "logstash-logs-%{+YYYY.MM.dd}"
  }
}
```
Just this should work, since we disable TLS listening by default. We ignore the `user` and `password` info, so no use adding that.